newIDE/app/src/MainFrame/AboutDialog.js

```diff
@@ -39,16 +39,23 @@ const styles = {
 // If you contributed to GDevelop but you're not in the list, please
 // send a Pull Request on GitHub or open an issue ;)
 const contributors = [
+  // Core team
+  {
+    name: 'Clément Pasteau',
+    description: 'CTO of the core development team',
```

Comment:
Uber nitpicking: Maybe just "Core development team"? (Avoiding titles in this list and focusing on the tasks done)